src/style.css

```
@@ -0,0 +1,124 @@
+@font-face {
+  font-family: Avenir-Book;
+  src: url(fonts/Avenir-Book.ttf);
+  font-style: normal;
+  font-weight: 300;
+}
+
+@font-face {
+  font-family: Avenir-Heavy;
+  src: url(fonts/Avenir-Heavy.ttf);
+  font-style: normal;
+  font-weight: 800;
+}
+
+.input {
+  font-family: "Avenir-Book", sans-serif;
```

```suggestion
  font-family: inherit;
```
Move font-family to the `html` tag selector and let inputs inherit it